examples/workflow-glsp/src/workflow-views.tsx

```diff
@@ -34,24 +38,32 @@ import { ActivityNode, Icon, TaskNode, WeightedEdge } from './model';
 const JSX = { createElement: snabbdom.svg };
 
 @injectable()
-export class TaskNodeView extends RectangularNodeView {
-    render(node: TaskNode, context: RenderingContext): VNode {
-        const rcr = this.getRoundedCornerRadius(node);
+export class TaskNodeView extends RoundedCornerNodeView {
+    protected renderWithoutRadius(node: Readonly<SShapeElement & Hoverable & Selectable>, context: RenderingContext): VNode {
+        const task = node as TaskNode;
+        const rcr = this.getRoundedCornerRadius(task);
         const graph = <g>
-            <rect class-sprotty-node={true} class-task={true}
-                class-automated={node.taskType === 'automated'}
-                class-manual={node.taskType === 'manual'}
-                class-mouseover={node.hoverFeedback} class-selected={node.selected}
```

Comment:
By removing `class-selected={node.selected}` we lose the selection feedback on task nodes

Reply:
Good catch! In the newly created manual nodes it works because it is considered in the rounded edge view but in the existing views loaded from the model file we lose it since I "optimized" those classes away. I re-added them for the non-corner case ;-)